### ISSUE TITLE
Add a confirmation when joining a zoom session as host

### ISSUE THREAD

**Issue by bcpierce00** [clowdr-app/clowdr-web-app#159]
Instead of directly joining Zoom, the "join as host" button should pop up a confirmation dialog warning people to use it carefully, with Proceed and Cancel buttons

**Comment by bcpierce00** [comment]
I left a comment about why this isn't totally straightforward in ZoomPanel.js.  Try again later.

**Comment by EdNutting** [comment]
What was this about? Is it worth consideration in the new UI?

**Comment by bcpierce00** [comment]
The issue was that a single item that started at 9AM and ran all day would prevent the program at a glance from progressing.  We should check for similar anomalies.

**Comment by EdNutting** [comment]
Err, err, your comment doesn't seem to match up to the issue, did you mean to post on a different one?

**Comment by bcpierce00** [comment]
Yes, I did -- I forget what issue number, but there was one about long program items messing up the scrolling of the program display.
